tests/grep_test.py

```diff
@@ -110,7 +110,7 @@ def _test_grep_color(file_config_files, capsys, *, args=()):
     expected = (
         '\033[1;34m{}\033[m'
         '\033[36m:\033[m'
-        'f'
+        '\033[35mf\033[m'
```

Comment:
can you split this out to a separate PR -- also this should accept both formats (it's unfortunately sensitive to the git version -- the version on my host doesn't highlight this purple)

Reply:
Done via https://github.com/asottile/all-repos/pull/219